libs.jstestdriver/external/jstestdriver-1.3.5-license.txt

```diff
@@ -0,0 +1,20 @@
+Name: JS Test Driver
+Version: 1.3.5
+License: Apache-2.0-REF
+Description: Google JS Test Driver for JS TDD development
+Origin: https://code.google.com/archive/p/js-test-driver/
```

Comment:
As the file is only used at compile time please also add a `Type: compile-time` header.